packages/vc-verification/src/models/credentials-types.ts

```diff
@@ -29,3 +33,15 @@ export interface RoleEIP191JWT {
   payload: RolePayload;
   eip191Jwt: string;
 }
+
+export interface CachedRoleCredential {
+  [key: string]: RoleEIP191JWT | VerifiableCredential<RoleCredentialSubject>;
+}
+
+export interface CachedRoleDefinition {
+  [key: string]: IRoleDefinitionV2;
+}
+
+export interface CachedDIDDocument {
+  [key: string]: IDIDDocument;
+}
```

Comment:
IMO, as these interfaces are only used as the type of the private cache implementations (as far as I can tell), I think this code should be removed and the type can be set directly in the cache implementations. My rationale for this is that "the best code is no code at all".
```suggestion
```